dogfood/Dockerfile

```
@@ -312,8 +312,24 @@ COPY --from=go /tmp/bin /usr/local/bin
 COPY --from=rust-utils /tmp/bin /usr/local/bin
 COPY --from=proto /tmp/bin /usr/local/bin
 
+# Configure Nix without sandboxing - https://github.com/NixOS/nix/issues/2636#issuecomment-455302745
+RUN addgroup --system nixbld \
+  && adduser coder nixbld \
+  && for i in $(seq 1 30); do useradd -ms /bin/bash nixbld$i &&  adduser nixbld$i nixbld; done \
+  && mkdir -m 0755 /nix && chown coder /nix \
```

Probably not mandatory, but Nix install instructions have `sudo install -d -m755 -o $(id -u) -g $(id -g) /nix`, which would translate to:

```suggestion
  && mkdir -m 0755 /nix && chown coder:coder /nix \
```